src/Compilers/Core/Portable/InternalUtilities/EnumUtilties.cs

```diff
@@ -52,5 +52,20 @@ internal static ulong ConvertEnumUnderlyingTypeToUInt64(object value, SpecialTyp
         {
             return (T[])Enum.GetValues(typeof(T));
         }
+
+#if DEBUG
```

Comment:
Use [Conditional("DEBUG")]

Reply:
`[Conditional]` cannot be applied to methods with a return value.

---
In reply to: [286169286](https://github.com/dotnet/roslyn/pull/35846#discussion_r286169286) [](ancestors = 286169286)